wordcloud/wordcloud.py

```diff
@@ -147,7 +149,7 @@ def single_color_func(word=None, font_size=None, position=None,
 
 
 class WordCloud(object):
-    """Word cloud object for generating and drawing.
+    r"""Word cloud object for generating and drawing.
```

Comment:
That doesn't change anything, does it?

Reply:
this string contains backslash character. so, we should use "r" prefix for treat backslashes as literal characters
(in second line of regexp parameter section, you use "r" prefix but it doesn't do anything because its in a string, so we should use "r" prefix for parent string)